README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
+```
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+#### Provision Storage
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample [rook-storageclass.yaml](demo/kubernetes/rook-storageclass.yaml).
 
-4. Use an S3 compatible client to create a bucket in the object store
+However, before we proceed, we need to specify the Ceph monitor endpoints. You can find them by running this line (you will need `jq`). This will generated a comma-separated list of monitor IPs and ports `6790` and place it into the `monitors` param of the `rook-storageclass.yaml`
 
-   ```bash
-   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ export MONS=$(kubectl -n rook get pod mon0 mon1 mon2 -o json|jq ".items[].status.podIP"|tr -d "\""|sed -e 's/$/:6790/'|paste -s -d, -)
+$ sed 's#INSERT_HERE#'$MONS'#' rook-storageclass.yaml | kubectl create -f -
+``` 
 
-5. List all buckets in the object store
+#### Consume the storage
 
-   ```bash
-   s3cmd ls --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   ```
+Now that rook is running and integrated with Kubernetes, we can create a sample app to consume the block storaged provisioned by rook. We will create the classic wordpress and mysql apps.
+Both these apps will make use of block volumes provisioned by rook.
 
-6. Upload a file to the newly created bucket
+Start mysql and wordpress:
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookObj
-   s3cmd put /tmp/rookObj --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f mysql.yaml
```

Comment:
mysql.yaml actually has the file extension .yml. i'm assuming you want to rename it for consistency